### PR TITLE
adapters/sqltimeout: Fix import order

### PR DESCRIPTION
**Pull request by andrewwormald**

### CODE REVIEW

File: adapters/sqltimeout/sqltimeout_test.go [hunk 1]
```diff
@@ -1,11 +1,12 @@
 package sqltimeout_test
 
 import (
-	"github.com/luno/workflow/adapters/sqltimeout"
 	"testing"
 
 	"github.com/luno/workflow"
 	"github.com/luno/workflow/adapters/adaptertest"
+
```
Comment (echarrod) [inline adapters/sqltimeout/sqltimeout_test.go:8]:
```suggestion
```

Comment (andrewwormald) [reply adapters/sqltimeout/sqltimeout_test.go:8]: These are separate modules 😄

Comment (andrewwormald) [reply adapters/sqltimeout/sqltimeout_test.go:8]: yes but `github.com/luno/workflow/adapters/adaptertest` is part of the `github.com/luno/workflow` mod and the line is automatically created on precommit hooks

Comment (echarrod) [reply adapters/sqltimeout/sqltimeout_test.go:8]: Sorry not sure I follow, so are you saying if you have:
```go
"github.com/luno/workflow"
"github.com/luno/workflow/adapters/adaptertest"
"github.com/luno/workflow/adapters/sqltimeout"
```
Then precommit changes it to:
```go
"github.com/luno/workflow"
"github.com/luno/workflow/adapters/adaptertest"

"github.com/luno/workflow/adapters/sqltimeout"
```
?

my suggestion was just to remove the empty line in between `"github.com/luno/workflow/adapters/adaptertest"` and `"github.com/luno/workflow/adapters/sqltimeout"`

**Comment by andrewwormald** [comment]
Closing for now as i will likely resolve with another MR and not need it to be independent

**Pull request closed without merging**